packages/block-editor-tools/README.md

```diff
@@ -72,8 +76,8 @@ This package is developed as part of the [Alley Scripts](https://github.com/alle
 
 You can contribute to this project in several ways:
 
-* Visit the main [Alley Scripts GitHub repo](https://github.com/alleyinteractive/alley-scripts) to [Open an issue](https://github.com/alleyinteractive/alley-scripts/issues/new) or submit PRs.
-* Alley employees can ask questions or ask for support in the [#javascript](https://alleyinteractive.slack.com/archives/C035Y7Q3X) channel in Slack.
+- Visit the main [Alley Scripts GitHub repo](https://github.com/alleyinteractive/alley-scripts) to [Open an issue](https://github.com/alleyinteractive/alley-scripts/issues/new) or submit PRs.
+- Alley employees can ask questions or ask for support in the [#javascript](https://alleyinteractive.slack.com/archives/C035Y7Q3X) channel in Slack.
```

Comment:
I realize you're updating this and it was already there, but please unlink `#javascript` here. I don't want a link to Alley's Slack in the README for a public repo.